specs/experimental/fault-proof/stage-one/bond-incentives.md

```diff
@@ -58,3 +64,71 @@ are located on the same level as honest claims, but situated to its right (i.e.
 An honest challenger can always successfully dispute any sibling claims not positioned to the right of an honest claim.
 The leftmost payoff rule encourages such disputes, ensuring only one claim is leftmost at correct depths.
 This claim will be the honest one, and thus bond rewards will be directed exclusively to honest claims.
+
+## Fault Proof Mainnet Incentives
+
+This section describes the specific bond incentives to be used for the Fault Proof Mainnet launch of the OP Stack fault
+proof system.
+
+### Gas Price Assumption
+
+FPM bonds are to assume a fixed 200 Gwei gas price.
+Future iterations of the fault proof may include a dynamic gas price calculation.
+For the moment, we suppose that the `GUARDIAN` address may account for increased average gas prices by updating the
+`OptimismPortal` contract to a new respected game type with a higher assumed gas price.
+
+### Bond Scaling
+
+FPM bonds are priced in the amount of gas that they are intended to cover.
+Bonds start at the very first depth of the game at a baseline of `400_000` gas.
+The `400_000` value is chosen as a deterrence amount that is approximately double the cost to respond at the top level.
+Bonds scale up to a value of `200_000_000` gas, a value chosen to cover approximately double the cost of a max-size
+Large Preimage Proposal.
+
+We use a multiplicative scaling mechanism to guarantee that the ratio between bonds remains constant.
+We determine the multiplier based on the proposed `MAX_DEPTH` of 73.
+We can use the formula `x = (200_000_000 / 400_000) ^ (1 / 73)` to determine that `x = 1.089`.
+At each depth `N`, the amount of gas charged is therefore `400_000 * (1.089 ^ N)`
+
+### Required Bond Formula
+
+Applying the [Gas Price Assumption](#gas-price-assumption) and [Bond Scaling](#bond-scaling) specifications, we have a
+`getRequiredBond` function:
+
+```python
+def get_required_bond(position):
+    assumed_gas_price = 200 gwei
+    base_gas_charged = 400_000
+    gas_charged = 400_000 * (1.089 ^ position.depth)
+    return gas_charged * assumed_gas_price
+```
+
+### DelayedWETH
+
+FPM introduces a contract `DelayedWETH` designed to hold the bonded ETH for each
+[Fault Dispute Game](./fault-dispute-game.md).
+`DelayedWETH` is an extended version of the standard `WETH` contract that introduces a delayed unwrap mechanism that
+allows the System Owner to function as a backstop in the case that a Fault Dispute Game would
+incorrectly distribute bonds.
+
+`DelayedWETH` is modified from `WETH` as follows:
+
+- `DelayedWETH` is a proxy contract owned by the System Owner.
+- `DelayedWETH` has a `startUnwrap` function that sets a timer and increments a mapping that keeps track of the
+`pending` unwrap amount for the address and the current timer for that address. Each address only has a single timer and
+the timer is always set to 7 days from the current timestamp whenever `startUnwrap` is called.
```

Comment:
Reading this section in isolation, it's unclear to me why this is chosen to be `7 days`

Reply:
Providing a concise explanation for this time period would make it much more comprehensible.